core/transcoder/utils.go

```diff
@@ -91,9 +91,19 @@ func handleTranscoderMessage(message string) {
 
 func createVariantDirectories() {
 	// Create private hls data dirs
+
+	err := os.RemoveAll(config.PublicHLSStoragePath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	err = os.RemoveAll(config.PrivateHLSStoragePath)
+	if err != nil {
+		log.Fatalln(err)
```

Comment:
Is it possible to fire `resetDirectories()` so the directory cleanup doesn't need to be duplicated?

Reply:
Moved the code to CleanupDirectory function in utils. What do you think about it?